### DISCUSSION TITLE
Tasmota smart meter reader gives no more extended values after power outage

### DISCUSSION THREAD

**Discussion by devinfluencer** [arendst/Tasmota#21079] [Q&A] [answered]
Hi,
I recently got a smart meter (ITron 3.HZ) and built a reader from a bitShake UART and a NodeMCU (ESP8266) with Tasmota 13.4.
Everything worked fine for some weeks, I had readings for Total consumption, Current consumption and meter ID.
Then we had a power outage and the smart meter reset and disabled the second row display again.
I entered the pin again and now can see the second row.
Tasmota however only gets readings for Total consumption, the other values are empty.
I tried a Tasmota reset and the sensor53 d1/d0 commands but to no avail.

What can I do?

Regards,
Thomas

**Comment by joba-1** [comment, marked as answer]
annoying, right? You need to explicitly switch the detailed info mode on (same itron menu as the pin input).

**Comment by devinfluencer** [comment]
Right you are, thank you!
Contrary to the meter's manual, it lost it's settings after 1 hr outage. The manual says 1 day...
